tests/test_fixtures.py

```diff
@@ -28,15 +28,15 @@ class TestTemplate1(Template):
     def package(self):
         return {'name': 'package.name',
                 'version': 'package.version',
-                'license': 'package.license'}
+                'pkg_license': 'package.pkg_license'}
 
     def image_layer(self):
         return {'diff_id': 'layer.diff',
                 'tar_file': 'layer.tarfile',
                 'packages': 'layer.packages'}
 
     def image(self):
-        return {'id': 'image.id',
+        return {'image_id': 'image.image_id',
```

Comment:
Good catch! Currently, the tests are *not* catching this. I'll see what's going on.